solutions/telemetry/README.md

```diff
@@ -0,0 +1,10 @@
+# Telemetry App Examples
+
+This folder contains a basic temperature telemetry Kafka app, in two flavors:
+
+- `console` contains a producer/consumer Kafka app in plain Java.
+- `quarkus` contains a producer/consumer Kafka app in Quarkus.
```

Comment:
We need to agree on how to call the apps and how to structure them.

Reply:
https://chat.google.com/room/AAAA1YpV2co/lncENFTMrOU